README.md

```diff
@@ -1 +1,172 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- JavaScript
+- Flexbox
+- Grid
+- (RWD) responsive web design using media query with mobile first principle
+- Animation and Transition
+- Use gitHub pages
+- Check the accessibility for this portfolio
+- Use the js DOM to close the mobile while clicking at the hamburger icon
+- Use the javascript objects and arrays to store the data for every project.
+
+### Tech Stack <a name="tech-stack"></a>
+
+> client
+
+- Html
+- JavaScript
+- Css
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[This project uses pure CSS, no framework.]**
+- **[Uses dynamic data insertion using javascript for the "featured musicians" section.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Link to the deployed project.
+
+- [Live Demo Link](https://yidnekachew-cmd.github.io/Module-1-Capstone/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+> In order to run this project you need:
+
+- Install code editor.
+- Install git.
+- Have github account.
+
+### Setup
+
+> Clone this repository to your desired folder:
+
+- git clone git@github.com:Yidnekachew-cmd/Module-1-Capstone.git
+
+### Usage
+
+> To run the project, do the following:
+
+- Install a liver server extension to your code editor and open it using the extension.
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Yidnekachew Kassahun**
+
+- GitHub: [Yidne](https://github.com/Yidnekachew-cmd)
+- Twitter: [Yidne](https://twitter.com/Yidnekassahun)
+- LinkedIn: [Yidne](https://www.linkedin.com/in/yidnekachew-kassahun-2b817a24b/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[I will add a contact section.]**
+- [ ] **[Create other pages like Program, Join and News]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please follow my github account. It helps a lot. Much Love.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank my friends Ermi, Nehmi, Aman a`nd besu.
```

Comment:
- [ ] [OPTIONAL] It would be nice if you gave credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).